tests/tracking-tests.yml

```diff
@@ -0,0 +1,362 @@
+!TestConfig
+clip_dir: tests/test-clips
+recording_tests:
+- !TestRecording
+  device: Wallaby
+  device_id: 956
+  filename: 551732.cptv
+  group: ORC
+  group_id: 160
+  rec_id: 551732
+  tracks:
+  - !TestTrack
+    confidence: 0.85
+    end: 43.22
+    end_pos:
+    - 43.11
+    - - 78
+      - 37
+      - 87
+      - 45
+    expected: true
+    opt_end: 43.22
+    opt_start: 0.0
+    start: 0.0
+    start_pos:
+    - 0.0
+    - - 75
+      - 36
+      - 85
+      - 46
+    tag: bird
+  - !TestTrack
+    confidence: 0.85
+    end: 43.22
+    end_pos:
+    - 43.11
+    - - 34
+      - 53
+      - 44
+      - 63
+    expected: true
+    opt_end: 43.22
+    opt_start: 8.11
+    start: 8.11
+    start_pos:
+    - 8.11
+    - - 10
+      - 30
+      - 19
+      - 38
+    tag: bird
+  - !TestTrack
+    confidence: 0.85
+    end: 20.67
+    end_pos:
+    - 20.56
+    - - 123
+      - 27
+      - 129
+      - 33
+    expected: true
+    opt_end: 20.67
+    opt_start: 3.0
+    start: 0
+    start_pos:
+    - 0
+    - - 4
+      - 91
+      - 14
+      - 101
+    tag: bird
+- !TestRecording
```

Comment:
Would probably be more manageable if each recording was independant in a separate file.   Then if you add a new test no 'old tests files' will need to change.